website/src/templates/event-post.js

```diff
@@ -75,11 +75,11 @@ export default function EventPost({
           When: {formatDate(frontmatter.date)}
         </Typography>
         <Typography variant="h6" className={classes.eventInfo}>
-          Where: {frontmatter.where}
+          Where: { frontmatter.where}
         </Typography>
       </div>
       <Typography variant="body1" className={classes.body}>
-        <div dangerouslySetInnerHTML={{ __html: html }} />
+        <span dangerouslySetInnerHTML={{ __html: html }} />
```

Comment:
Was getting console errors that you can't nest a div inside a <p> tag which, yes.